styles.css

```diff
@@ -260,6 +266,106 @@ body {
   transition: all 0.8s cubic-bezier(0.075, 0.82, 0.165, 1);
 }
 
+/* project details popup start */
+#project-details-modal {
+  position: fixed;
+  width: 100%;
+  background-color: rgba(3, 3, 3, 0.596);
+  backdrop-filter: blur(6px);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 20;
+  padding: 60px 16px 40px 16px;
+  display: none;
+}
+
+#popup-details-card {
+  height: calc(100% + 2px);
+  padding: 16px;
+  background-color: #fff;
+  overflow-y: scroll;
+  border: 1px solid #dfe1e6;
+}
+
+#close {
+  font-size: 24px;
+  color: #67798e;
+  position: absolute;
+  padding: 8px;
+  right: 0;
+  cursor: pointer;
+}
+
+#close:hover i,
+#close:active i {
+  color: red;
+}
+
+.popup-heading {
+  position: relative;
+}
+
+#popup-heading {
+  font-family: 'Roboto', sans-serif;
```

Comment:
- [ ] Great work on designing the header for your new popup! :+1: However, as per the project specifications, it is necessary to use the same font family that is depicted in the Figma template. Therefore, I suggest you switch the font to `'Crete Round', sans-serif`